app/workers/queue_listener.rb

```diff
@@ -28,6 +30,10 @@ def create_action(params)
     AkActionCreator.create_action(params[:params][:slug], params[:params][:body])
   end
 
+  def update_resource(params)
+    Ak::Updater.update(uri: params[:uri], body: params[:params])
```

Comment:
It would be nice if the naming scheme between `Ak::Updater` and `AkPageCreator` was consistent